blocks/byline-block/index.story.mdx

```diff
@@ -0,0 +1,27 @@
+import { Meta, Story, Canvas } from "@storybook/addon-docs";
+
+import Byline from "./features/byline/default";
+
+<Meta title="Blocks/Byline" component={Byline} />
+
+# Byline
+
+The `Byline` component reads from globalContent and returns the author(s) in a comma delimited format.
+
+## Usage
```

Comment:
We should not be documenting usage for blocks as the only consumer of blocks are PageBuilder